Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -265,6 +266,13 @@ public void Initialize(TelemetryConfiguration configuration)
             this.telemetryClient = new TelemetryClient(configuration);
             this.telemetryClient.Context.GetInternalContext().SdkVersion = SdkVersionUtils.GetSdkVersion("web:");
 
+            this.DisableTrackingProperties = configuration.EvaluateExperimentalFeature("deferRequestTrackingProperties");
```

Comment:
"deferRequestTrackingProperties" - please move this to a const in shared proj. I believe its duplicated in qp and request tracking ..